components/Cards/README.md

```diff
@@ -281,7 +281,7 @@ Swift
 ```swift
   card.isAccessibilityElement = true
   card.accessibilityLabel = "Location (userLocation.name) is popular with users " +
-    "who enjoy \(userLocation.popularActivityMatchingUserProfile(userProfile))"
+    "who enjoy (userLocation.popularActivityMatchingUserProfile(userProfile))"
```

Comment:
Nit: I believe we still need the "\" and there should be one on the line above.

Reply:
File a bug please!

Reply:
Should we roll-back this commit for the release? Rather than making our docs look even more wrong once they're published?

Reply:
Fixed and regenerated.